drivers/storage/portworx/testspec/prometheusRule.yaml

```diff
@@ -148,15 +148,15 @@ spec:
       annotations:
         description: Pool {{$labels.POOL}} from node {{$labels.instance}}, from Portworx cluster {{$labels.clusterid}} successfully expanded.
         summary: Portworx pool {{$labels.POOL}} successfully expanded.
-      expr: px_alerts_poolexpandsuccessful == 1
+      expr: px_alerts_poolexpandsuccessful > 1
```

Comment:
Just for learning/sanity check's sake: my initial reaction would be "shouldn't this be >= 1 instead"? I know the error I got said ">", not ">=" so this is correct to fix the tests, I'm just curious if this is correct (or if the point is that it's a test file and if it makes sense doesn't matter)

Reply:
from the new config file in `bin/config` it's `>1` I believe that's the source of truth.